### ISSUE TITLE
Click the market for more infomation

### ISSUE THREAD

**Issue by AbeHandler** [TheLens/land-records#96]
That is confusing. if you see the info box, you are hovered over the marker. so just say "click for more information"   i just tried to move my mouse to the marker./

**Comment by ThomasThoren** [comment]
This is how it used to read, but I found that more confusing. When the popup text said, "Click for more information," that seemed to suggest moving your mouse to the text in the popup and then clicking. This of course doesn't work since the popup closes once you move your mouse away from the marker.

If you mouse to a marker, hover over it and then read, "Click the marker for more information," that seems about as clear as it can get don't you think?

**Comment by ThomasThoren** [comment]
@myersnews, what do you think?

**Comment by myersnews** [comment]
Yeah, this issue came from me trying to click on the info box back when it said "click for more information." You can do an informal test in the office and see what people say.
